docs/spec/generics.rst

```diff
@@ -458,8 +450,43 @@ Also consider the following example::
   class MyDict(Mapping[str, T]):
       ...
 
-In this case MyDict has a single parameter, T.
+In this case ``MyDict`` has a single parameter, ``T``.
```

Comment:
For clarity, this should probably say "type parameter" rather than just "parameter".